apps/03TicTacToe.js

```diff
@@ -24,23 +24,86 @@ function printBoard() {
 }
 
 function horizontalWin() {
-  // Your code here
+
+//  Your code here
+  if ((board[0][0] === 'X' && board[0][1] === 'X' && board[0][2] === 'X' )
+   || (board[1][0] === 'X' && board[1][1] === 'X' && board[1][2] === 'X')
+   || (board[2][0] === 'X' && board[2][1] === 'X' && board[2][2] === 'X')
+   || (board[0][0] === 'O' && board[0][1] === 'O' && board[0][2] === 'O')
+   || (board[1][0] === 'O' && board[1][1] === 'O' && board[1][2] === 'O')
+   || (board[2][0] === 'O' && board[2][1] === 'O' && board[2][2] === 'O')) {
+      console.log(playerTurn + " !!! WINS!!!    ...Resetting Game");
+
+        board = [
+          [' ', ' ', ' '],
+          [' ', ' ', ' '],
+          [' ', ' ', ' ']
+        ];
+return true;
+  } else
+  return false;
 }
 
 function verticalWin() {
-  // Your code here
+        if  ((board[0][0] === 'X' && board[1][0] === 'X' && board[2][0] === 'X' )
+          || (board[0][1] === 'X' && board[1][1] === 'X' && board[2][1] === 'X')
+          || (board[0][2] === 'X' && board[1][2] === 'X' && board[2][2] === 'X')
+          || (board[0][0] === 'O' && board[1][0] === 'O' && board[2][0] === 'O')
+          || (board[0][1] === 'O' && board[1][1] === 'O' && board[2][1] === 'O')
+          || (board[0][2] === 'O' && board[1][2] === 'O' && board[2][2] === 'O'))  {
+      console.log(playerTurn + " !!! WINS!!!    ...Resetting Game.");
+
+        board = [
+          [' ', ' ', ' '],
+          [' ', ' ', ' '],
+          [' ', ' ', ' ']
+        ];
+  return true;
+  } else
+  return false;
 }
 
 function diagonalWin() {
   // Your code here
+  if  ((board[0][0] === 'X' && board[1][1] === 'X' && board[2][2] === 'X' )
+    || (board[0][2] === 'X' && board[1][1] === 'X' && board[2][0] === 'X')
+    || (board[0][0] === 'O' && board[1][1] === 'O' && board[2][2] === 'O')
+    || (board[0][2] === 'O' && board[1][1] === 'O' && board[2][0] === 'O')) {
+  console.log(playerTurn + " !!! WINS!!!    ...Resetting Game.");
+
+   board = [
```

Comment:
Try putting your resetting 'board' logic in a function to make your code more dry.